### PR TITLE
Update Sidecar Docs with Latest Version

### PR DESCRIPTION
**Pull request by albertov19**
### Description

Add changes to update Sidecar docs to the latest sidecar version

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [x] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [x] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [x] If this page requires a disclaimer, I have added one

### Corresponding PRs

https://github.com/PureStake/moonbeam-docs-cn/pull/188

### CODE REVIEW

File: builders/get-started/eth-compare/tx-fees.md [hunk 1]
```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
 
 === "EIP-1559"
     ```
-    Transaction Fee =（Base Fee + Max Priority Fee Per Gas) * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    Gas Price = BaseFee + MaxPriorityFeePerGas < MaxFeePerGas ? 
+                BaseFee + MaxPriorityFeePerGas : 
+                MaxFeePerGas;
+    Transaction Fee = (Gas Price * Transaction Weight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}
```
Comment (eshaben) [inline builders/get-started/eth-compare/tx-fees.md:78]:
```suggestion
    Gas Price = Base Fee + Max Priority Fee Per Gas < Max Fee Per Gas ? 
                Base Fee + Max Priority Fee Per Gas : 
                Max Fee Per Gas;
    Transaction Fee = (Gas Price * Transaction Weight) / {{ networks.moonbase.tx_weight_to_gas_ratio }}
```

all of the other variables have a space between the words so I would change these just to be consistent

File: builders/get-started/eth-compare/tx-fees.md [hunk 2]
```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
...
+=== "Moonriver"
+    ```
+    86298000
+    ```
+=== "Moonbase Alpha"
+    ```
+    250000000
+    ```
+
+The values of `Gas Price`, `MaxFeePerGas` and `MaxPriorityFeePerGas` for the applicable transaction types can be read from the block JSON object according to the structure described in [the Sidecar API page](/builders/build/substrate-api/sidecar/#evm-fields-mapping-in-block-json-object){target=_blank}, also truncated and reproduced below:
```
Comment (eshaben) [inline builders/get-started/eth-compare/tx-fees.md:104]:
```suggestion
The values of `Gas Price`, `Max Fee Per Gas` and `Max Priority Fee Per Gas` for the applicable transaction types can be read from the block JSON object according to the structure described in [the Sidecar API page](/builders/build/substrate-api/sidecar/#evm-fields-mapping-in-block-json-object){target=_blank}, also truncated and reproduced below: 
```

File: builders/get-started/eth-compare/tx-fees.md [hunk 3]
```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
...
+The values of `Gas Price`, `MaxFeePerGas` and `MaxPriorityFeePerGas` for the applicable transaction types can be read from the block JSON object according to the structure described in [the Sidecar API page](/builders/build/substrate-api/sidecar/#evm-fields-mapping-in-block-json-object){target=_blank}, also truncated and reproduced below: 
 
 === "EIP1559"
-    |        EVM Field         |                               Block JSON Field                                |
-    |:------------------------:|:-----------------------------------------------------------------------------:|
-    | Max priority fee per gas | `extrinsics[extrinsic_number].args.transaction.eip1559.maxPriorityFeePerGas` |
+    |      EVM Field       |                               Block JSON Field                               |
+    |:--------------------:|:----------------------------------------------------------------------------:|
+    |     MaxFeePerGas     |     `extrinsics[extrinsic_number].args.transaction.eip1559.maxFeePerGas`     |
+    | MaxPriorityFeePerGas | `extrinsics[extrinsic_number].args.transaction.eip1559.maxPriorityFeePerGas` |
```
Comment (eshaben) [inline builders/get-started/eth-compare/tx-fees.md:110]:
```suggestion
    |     Max Fee Per Gas     |     `extrinsics[extrinsic_number].args.transaction.eip1559.maxFeePerGas`     |
    | Max Priority Fee Per Gas | `extrinsics[extrinsic_number].args.transaction.eip1559.maxPriorityFeePerGas` |
```

File: builders/get-started/eth-compare/tx-fees.md [hunk 4]
```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
...
 === "EIP2930"
-    |      EVM Field       |                            Block JSON Field                             |
-    |:--------------------:|:-----------------------------------------------------------------------:|
-    |      Gas price       |    `extrinsics[extrinsic_number].args.transaction.eip2930.gasPrice`    |
+    | EVM Field |                         Block JSON Field                         |
+    |:---------:|:----------------------------------------------------------------:|
+    | Gas Price | `extrinsics[extrinsic_number].args.transaction.eip2930.gasPrice` |
 
-The `Base Fee`, introduced in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559){target=_blank}, is a value set by the network itself. The `Base Fee` for `EIP1559` type transactions is currently static on Moonbeam networks and has the following assigned value:
+The `BaseFee`, introduced in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559){target=_blank}, is a value set by the network itself. The `BaseFee` for `EIP1559` type transactions is currently static on Moonbeam networks and has the following assigned value:
```
Comment (eshaben) [inline builders/get-started/eth-compare/tx-fees.md:122]:
```suggestion
The `Base Fee`, introduced in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559){target=_blank}, is a value set by the network itself. The `Base Fee` for `EIP1559` type transactions is currently static on Moonbeam networks and has the following assigned value:
```

File: builders/get-started/eth-compare/tx-fees.md [hunk 5]
```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
...
 
 === "Moonbeam"
     | Variable |  Value   |
     |:--------:|:--------:|
-    | Base fee | 100 Gwei |
+    | BaseFee  | 100 Gwei |
```
Comment (eshaben) [inline builders/get-started/eth-compare/tx-fees.md:127]:
```suggestion
    | Base Fee  | 100 Gwei |
```

File: builders/get-started/eth-compare/tx-fees.md [hunk 6]
```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
...
 
 === "Moonriver"
     | Variable | Value  |
     |:--------:|:------:|
-    | Base fee | 1 Gwei |
+    | BaseFee  | 1 Gwei |
```
Comment (eshaben) [inline builders/get-started/eth-compare/tx-fees.md:132]:
```suggestion
    | Base Fee  | 1 Gwei |
```

File: builders/get-started/eth-compare/tx-fees.md [hunk 7]
```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
...
 
 === "Moonbase Alpha"
     | Variable | Value  |
     |:--------:|:------:|
-    | Base fee | 1 Gwei |
+    | BaseFee  | 1 Gwei |
```
Comment (eshaben) [inline builders/get-started/eth-compare/tx-fees.md:137]:
```suggestion
    | Base Fee  | 1 Gwei |
```

File: variables.yml [hunk 8]
```diff
@@ -222,7 +222,7 @@ networks:
       collective_tech_committee: '0x000000000000000000000000000000000000080f'
       collective_treasury: '0x0000000000000000000000000000000000000810'
     substrate_api_sidecar:
-      stable_version: 11.3.9
+      stable_version: 14.1.1
```
Comment (eshaben) [inline variables.yml:225]: only MBA needs to be updated?

**Review by eshaben — CHANGES_REQUESTED** [review]

**Comment by albertov19** [comment]
@eshaben thanks for the review 😃 - I've applied the feedback (also in the CN repo)

File: variables.yml (hunk 8 above)
Comment (albertov19) [reply variables.yml:225]: Good catch!

File: builders/get-started/eth-compare/tx-fees.md [hunk 9]
```diff
@@ -72,61 +72,85 @@ To calculate the fee incurred on a Moonbeam transaction sent via the Ethereum AP
 
 === "EIP-1559"
     ```
-    Transaction Fee =（Base Fee + Max Priority Fee Per Gas) * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    Gas Price = Base Fee + Max Priority Fee Per Gas< Max Fee Per Gas ?
```
Comment (eshaben) [inline builders/get-started/eth-compare/tx-fees.md:75]:
```suggestion
    Gas Price = Base Fee + Max Priority Fee Per Gas < Max Fee Per Gas ? 
```

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**